easybuild/tools/github.py

```diff
@@ -324,6 +325,22 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
         # make sure path exists, create it if necessary
         mkdir(path, parents=True)
 
+    develop = download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch='develop', path=path)
+    if not os.path.isdir(develop):
+        raise EasyBuildError("Downloading of develop branch failed: not found in %s" % develop)
```

Comment:
use `, develop` rather than `% develop`, there's a difference (with the former, the message will only be templated when debug logging is enabled, which is slightly faster)